src/logic/calculate.js

```diff
@@ -0,0 +1,30 @@
+import operate from './operate';
+
+function calculate(data, buttonName) {
```

Comment:
@Alexoid1, Good job so far.

According to the instructions on the pathwright, we need to pass an object as an argument.

The argument passed to the `function calculate` should be an object, we can achieve this by using destructuring.
```
function calculate(data, buttonName) {
  let { total, next, operation } = data;
.....
}
```

![image](https://user-images.githubusercontent.com/31897434/104987203-6af22800-5a15-11eb-881b-0d3b69374ddb.png)

https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/